Reaqtor/Core/Engine/Tests.Reaqtor.QueryEngine/InnerSubjectTests.cs

```diff
@@ -68,7 +68,7 @@ public void Engine_Subject_AsIObservable()
 
             var xs = new List<int>();
 
-            var s = (ISubscription)((IObservable<int>)subject).Subscribe(xs.Add);
+            var s = (ISubscription)subject.Subscribe(xs.Add);
```

Comment:
This is another one where I think the cast was drawing attention to how the test works. I'd leave it in and add a pragma